manage.py

```diff
@@ -1,4 +1,4 @@
-from goodpackagenamehere.control import cli
+from ner_trainer.control import cli
```

Comment:
We've decided not to use 'ner_trainer' anymore. 'Goodpackagename' was just like an stub before we come up with a brand new name for that stuff. As @dchaplinsky thought: it should have been a motivation to anybody who will change it.
(not an objection, just an eat for thoughts)